### PR TITLE
Fix using Python's runtime module directly

### PR DESCRIPTION
**Pull request by helderco**
Can be useful for debugging.

### CODE REVIEW

File: sdk/python/runtime/main.go [hunk 1]
```diff
@@ -77,17 +81,21 @@ var tplMain string
 // The others were built to be composable and chainable to facilitate the
 // creation of extension modules (custom SDKs that depend on this one).
 type PythonSdk struct {
+    // Directory with the Python SDK source code
 	SDKSourceDir  *Directory
+
+    // List of patterns to allways include when loading Python modules
```
Comment (jedevc) [inline sdk/python/runtime/main.go:87]: nit :scream: - the indentation here should be tab, `go fmt` should fix it.

Comment (helderco) [reply sdk/python/runtime/main.go]: It's always the same with me, because I don't have the IDE do it automatically 😅

**Review by jedevc — APPROVED** [review]

**Comment by helderco** [comment]
Is this happening more often?
```
failed to compute cache key: failed to get state for index 0 on copy /runtime/template/runtime.py /runtime
```

**Comment by jedevc** [comment]
@helderco yup, this has started happening a *lot* more - we're trying to investigate, see https://github.com/dagger/dagger/issues/7020

**Pull request merged**